config/config.go

```diff
@@ -165,7 +174,29 @@ type jsonConfig struct {
 	Monitor    jsonSection      `json:"monitor,omitempty"`
 	Allocator  jsonSection      `json:"allocator,omitempty"`
 	Informer   jsonSection      `json:"informer,omitempty"`
-	Sharder    jsonSection      `json:"sharder,omitempty"`
+}
+
+func (jcfg *jsonConfig) getSection(i SectionType) jsonSection {
+	switch i {
+	case Consensus:
```

Comment:
no `Cluster` section here?

Reply:
Fixed (it's not used and I forgot).

Reply:
No, actually there is not a section for Clusterbecause it's a *json.RawMessage and it's handled differently. So this was correct.